lib/ansible/modules/cloud/ovirt/ovirt_host_networks.py

```diff
@@ -180,7 +180,7 @@ def update_address(self, attachments_service, attachment, network):
                 if not equal(network.get('gateway'), ip.ip.gateway):
                     ip.ip.gateway = network.get('gateway')
                     changed = True
-                if not equal(network.get('prefix'), int(ip.ip.netmask) if ip.ip.netmask else None):
+                if not equal(network.get('prefix'), sum([bin(int(x)).count('1') for x in ip.ip.netmask.split('.')]) if ip.ip.netmask else None):
```

Comment:
typecasting using int() will still raise ValueError

Reply:
I think, first validating ip with 

https://github.com/ansible/ansible/blob/75a34f666826be988b25df41cb72bcd5d1428736/lib/ansible/module_utils/network/common/utils.py#L354

Reply:
Please look again..  It's a for loop, that splits the netmask into octets between the ".", converts to it a int, then gets the binary count.

ie, "255.255.255.0" -> [ "255", "255", "255", "0"]  which is then converted from a string to an integer, and then bin counts the 1's in the int, which returned to the sum function as 4 numbers (ie, [ 8,8,8,0] )

I have been using this code, and it does work correctly.

Reply:
Got it. Add attribution link for Stackoverflow answer's link.